virtcontainers/container.go

```diff
@@ -1185,191 +1152,3 @@ func (c *Container) detachDevices() error {
 	}
 	return nil
 }
-
-func (c *Container) addResources() error {
```

Comment:
Please squash this commit with the next one. We want in the same commit the part where you remove from container to apply onto sandbox.

Reply:
fixed